curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660ef19b95d3308e7dd31bb6.md

```diff
@@ -20,10 +20,11 @@ Print the value of the `character` variable to the console. Then, click the "Con
 
 # --hints--
 
-You should declare a variable `character` and assign it a value.
+The `character` variable declaration should not be changed.
 
 ```js
-assert.match(__helpers.removeJSComments(code), /let\s+character\s*=\s*[^;]+;/);
+assert.isDefined(character);
+assert.isNotEmpty(character);
```

Comment:
please check that the value was not changed instead of just checking that it is not empty